src/scanners/css.js

```diff
@@ -17,16 +17,22 @@ export default class CSSScanner extends BaseScanner {
           if (ast && ast.stylesheet && ast.stylesheet.rules) {
             var rules = ignorePrivateFunctions(_rules);
 
-            for (let cssRule in rules) {
+            for (let cssRuleFunc in rules) {
               this._rulesProcessed++;
 
-              for (let rule of ast.stylesheet.rules) {
-                if (rule.type === 'comment') {
+              for (let cssRule of ast.stylesheet.rules) {
+                if (cssRule.type === 'comment') {
                   continue;
                 }
 
                 this.validatorMessages = this.validatorMessages.concat(
-                  rules[cssRule](rule));
+                  rules[cssRuleFunc](cssRule, cssRule.position.source,
+                    {
+                      startLine: cssRule.position.start.line,
+                      startColumn: cssRule.position.start.column,
+                    }
```

Comment:
You're missing the default object here. Calling this without passing the optional args object would result in an error.

Contrast: http://jsbin.com/gumifo/edit?js,console,output 

with this: http://jsbin.com/pimibi/edit?js,console,output